src/tron/tronEngine.ts

```diff
@@ -1329,12 +1334,13 @@ export class TronEngine extends CurrencyEngine<TronTools> {
     return edgeTransaction
   }
 
-  getDisplayPrivateSeed(): string {
-    return this.walletInfo.keys?.tronMnemonic ?? this.walletInfo.keys?.tronKey
+  getDisplayPrivateSeed(privateKeys: JsonObject): string | null {
+    const tronPrivateKeys = asTronPrivateKeys(privateKeys)
+    return tronPrivateKeys.tronMnemonic ?? tronPrivateKeys.tronKey
```

Comment:
I think this structure would make sense for the other plugins where we currently return "" if the mnemonic is missing. We should always return something.

Reply:
What are you asking for here? This actuall will always return a string.

Reply:
The `getDisplayPrivateSeed` should never return `null` or `''` if it can help it. Tron is doing things the right way, since it always returns something useful, but there are other plugins in this repo where we do return `''` as a fallback. We should find those and fix them.